### PR TITLE
Update preprocessing script for spine-generic with new naming convention

### PR DESCRIPTION
**Pull request by sandrinebedard**
## Description

This PR updates the preprocessing script with the new naming convention of the derivatives (see issue https://github.com/spine-generic/data-multi-subject/issues/157).

It also creates the `labels_softseg_bin`, binarized softseg labels.

It add reorientation to RPI of all the labels to make sure they are in RPI.

### CODE REVIEW

File: processing_spine_generic/process_data.sh [hunk 1]
```diff
@@ -348,20 +355,20 @@ for file_path in "${inc_contrasts[@]}";do
   elif [[ $file_path == *"T2star"* ]];then
       contrast_seg="t2s"
       contrast=contrast_seg
-  elif [[ $file_path == *"flip-2_mt-off_MTS"* ]];then
+  elif [[ $file_path == *"flip-2_mt-off_space-other_MTS"* ]];then
       contrast_seg="t1"
-      contrast="flip-2_mt-off_MTS"
-  elif [[ $file_path == *"flip-1_mt-on_MTS"* ]];then
+      contrast="flip-2_mt-off_space-other__MTS"
```
Comment (valosekj) [inline processing_spine_generic/process_data.sh:360]: typo in double `_`

```suggestion
      contrast="flip-2_mt-off_space-other_MTS"
```

File: processing_spine_generic/process_data.sh [hunk 2]
```diff
@@ -501,26 +511,37 @@ for file_path in "${inc_contrasts[@]}";do
...
+  # REORIENT all:
   # Move json files of derivatives
   rsync -avzh "${PATH_DATA}/derivatives/labels/${SUBJECT}/${fileseg}.json" $PATH_DATA_PROCESSED_CLEAN/derivatives/labels/${SUBJECT}/${fileseg}.json
   rsync -avzh "${PATH_DATA}/derivatives/labels_softseg/${SUBJECT}/${filesoftseg}.json" $PATH_DATA_PROCESSED_CLEAN/derivatives/labels_softseg/${SUBJECT}/${filesoftseg}.json
+  rsync -avzh "${PATH_DATA}/derivatives/labels_softseg/${SUBJECT}/${filesoftseg}.json" $PATH_DATA_PROCESSED_CLEAN/derivatives/labels_softseg_bin/${SUBJECT}/${fileseglabel_bin}.json
+ 
   # Move disc labels into cleaned derivatives
-  rsync -avzh $PATH_DATA_PROCESSED/${SUBJECT}/${file_path}_seg_labeled_discs.nii.gz $PATH_DATA_PROCESSED_CLEAN/derivatives/labels/${SUBJECT}/${file_path}_discs.nii.gz
-
+  # TODO: ADD IF STATEMENT FOR T1 and T2w label-discs_dlabel instead
```
Comment (valosekj) [inline processing_spine_generic/process_data.sh:529]: The if statement is already added; we can probably remove the TODO?

```suggestion
```

**Review by valosekj — APPROVED** [review]
Thank you @sandrinebedard! I haven't run the script, but the changes look good. I just left two minor comments.

**Comment by sandrinebedard** [comment]
Thanks Jan!

**Pull request merged**